flink-libraries/flink-table/src/main/scala/org/apache/flink/table/runtime/join/NonWindowJoin.scala

```diff
@@ -18,7 +18,7 @@
 package org.apache.flink.table.runtime.join
 
 import org.apache.flink.api.common.functions.FlatJoinFunction
-import org.apache.flink.api.common.state.{MapState, MapStateDescriptor, ValueState, ValueStateDescriptor}
+import org.apache.flink.api.common.state._
```

Comment:
Do you think we can extract a `CoProcessFunctionWithCleanupState` which can be share all operation of two sides？just a suggestion, I'm fine keep current state.

Reply:
Good idea! I will add a `CoProcessFunctionWithCleanupState`.